docarray/array/doc_vec/doc_vec.py

```diff
@@ -96,11 +106,17 @@ def __init__(
         docs: Sequence[T_doc],
         tensor_type: Type['AbstractTensor'] = NdArray,
     ):
+
+        if not hasattr(self, 'doc_type') or self.doc_type == AnyDoc:
+            raise TypeError(
+                f'{self.__class__.__name__} does not precise a doc_type. You probably should do'
```

Comment:
```suggestion
                f'{self.__class__.__name__} does not precise a doc_type. You need to do'
```
If it fails then it is not "probably", right?